.env

```diff
@@ -1,4 +1,4 @@
 REACT_APP_GOOGLE_CLIENT_ID=817921738258-odg272ueou0t9q4vc15rrjbevh91uupn.apps.googleusercontent.com
 REACT_APP_API_URL=http://localhost:8080/api
 REACT_APP_OAUTH2_REVOKE_API=https://accounts.google.com/o/oauth2/revoke
-REACT_APP_MSW_MOCKING_ENABLED=false
+REACT_APP_MSW_MOCKING_ENABLED=true
```

Comment:
add this variable to .env too, this way we don't have to modify prod set up anymore